src/Entities/User.php

```diff
@@ -46,10 +46,14 @@ public function getIdentity(string $type)
      */
     public function getIdentities(): array
     {
-        if (! array_key_exists('identities', $this->attributes) || ! is_array($this->attributes['identities'])) {
-            $this->attributes['identities'] = model(UserIdentityModel::class)
-                ->where('user_id', $this->id)
-                ->findAll();
+        if (
+            ! array_key_exists('identities', $this->attributes)
+            || ! is_array($this->attributes['identities'])
+        ) {
+            /** @var UserIdentityModel $identityModel */
+            $identityModel = model(UserIdentityModel::class);
+
+            $this->attributes['identities'] = $identityModel->getIdentities($this->id);
```

Comment:
I realize this preexists you but... I feel fairly certain that relations should not go in `$attributes`. I've been down this path and it can cause a lot of confusion around what is in an Entity. 

My usually solution is a private/protected property with getter/setter methods; this also helps with static analysis.

Reply:
Personally, I dislike magic properties. Preventing static analysis, and refactoring by IDE.
It is difficult to know what properties there.

You say so, it seems better to replace it with a private property with getter/setter methods.
I created Issue #65.

Reply:
Thanks! We'll get to that, shouldn't be difficult.